api/README.md

```diff
@@ -0,0 +1,123 @@
+
+The ParallelCluster API specification, defined in the `spec/` directory, is available in two different formats:
+[OpenAPI](https://swagger.io/specification/) and [Smithy](https://awslabs.github.io/smithy/).
+
+The OpenAPI definition can be found in the `spec/openapi/ParallelCluster.openapi.yaml` file while the Smithy model is
+defined in the `spec/smithy` directory.
+
+The OpenAPI model can be automatically generated from the Smithy one by using the following Gradle task:
+```bash
+./gradlew buildSmithyModel
+```
+
+Additional Gradle tasks are available in order to streamline the development of the API:
+```bash
+./gradlew tasks
+...
+OpenAPI Tools tasks
+-------------------
+openApiGenerate - Generate code via Open API Tools Generator for Open API 2.0 or 3.x specification documents.
+openApiGenerators - Lists generators available via Open API Generators.
+openApiMeta - Generates a new generator to be consumed via Open API Generator.
+openApiValidate - Validates an Open API 2.0 or 3.x specification document.
+
+ParallelCluster API tasks
+-------------------------
+buildSmithyModel - Build Smithy model and generate the OpenAPI spec file
+generatePythonClient - Generate a Python client using Open API Tools Generator.
+generatePythonServer - Generate a Python server using Open API Tools Generator.
+redoc - Generate a standalone html page with the redoc documentation of the API
+swaggerUI - Run a Docker container hosting a Swagger UI with the ParallelCluster API specs.
+```
+
+### OpenAPI Documentation Tools
+
+From the OpenAPI model, a visual documentation can be produced, in either the [Swagger UI](https://swagger.io/tools/swagger-ui/)
+or the [ReDoc](https://github.com/Redocly/redoc) format.
+
+To run local server hosting the Swagger UI with live reloading enabled run the following command:
+`./gradlew swaggerUI`.
+
+You can then review the documentation by opening a browser to http://0.0.0.0:8080.
+To stop the Docker container exposing the Swagger UI run `docker stop pcluster-swagger-ui`.
+
+To generate a standalone HTML page with the API documentation in the ReDoc format run the following command:
+`./gradlew redoc`. Once compiled the file will be available at `spec/openapi/ParallelCluster.openapi.redoc.html`.
+
+### Generating the API Server Stub and API Client
+
+The [OpenAPI Generator](https://openapi-generator.tech/) tool can be used to generate the client and the server
+applications form the OpenAPI specification file.
+
+In order to generate the server stub in Python language the [python-flask](https://openapi-generator.tech/docs/generators/python-flask)
+generator can be invoked by running: `./gradlew generatePythonServer`. The code is generated under the 
+`generated/python-server` directory.
+
+In order to generate the client in Python language the [python](https://openapi-generator.tech/docs/generators/python)
+generator can be invoked by running: ``./gradlew generatePythonClient`. The code is generated under the
```

Comment:
extra \` character before `.gradlew` and at the end of the sentence.

Reply:
fixed